source/online_usecases/subcellular_level/subcellular_app/subcellular_app.rst

```diff
@@ -0,0 +1,265 @@
+
+It was designed to reach several objectives related to major restrictions
+of currently available software tools, such as the lack of integration
+with existing biological data relevant for modeling and low compatibility
+of different types of models.
+
+General info
+=====================
+
+The subcellular application was designed as a hub web based environment
+for creation and simulation of reaction-diffusion models integrated with
+the molecular repository.
+It allows also importing, combining and simulation of existing models
+expressed with `BNGL`_ and `SBML`_ languages.
+
+Two types of models are supported: rule-based models convenient and
+computationally efficient for modeling of big protein signaling complexes
+and chemical reaction network models.
+
+The subcellular application is integrated with a number of solvers for
+reaction-diffusion systems of equations. It supports simulation of spatially
+distributed systems using STEPS (stochastic engine for pathway simulation) –
+which provides spatial stochastic and deterministic solvers for simulation
+of reactions and diffusion on tetrahedral meshes.
+
+Application as well provides a number of facilities for a visualization
+of models geometry and results of simulations.
+
+The molecular repository is a publicly available database of biological
+information relevant for brain molecular network modeling.
+
+It accomodates several types of biological information which are not available
+from existing public databases, such as concentrations of proteins in
+different subcellular compartments of neuronal and glial cells, kinetic data
+on proteins interactions specific for brain and synaptic signaling and
+plasticity, data on molecules mobility.
+
+The repository is integrated with the subcellular application. They share
+the same set of entities described by BioNetGen expressions. The molecular
+repository can be queried from the subcellular application and the results
+of query can be added to a molecular network model.
+
+.. image:: images/model_reactions.png
+   :width: 800 px
+
+Usage
+=====================
+
+#.  Open subcellular application by using link: `SubcellularApp`_
+#.  Create a new model or load/import an existing one
+
+    * To load a public model click an arrow to expand a list of public models
+      in `Subcellular models repository`, choose one, then click `Load`
+    * To import `BNGL`_ or `SBML`_ (`Atomizer`_ is used for latter) model click
+      `Import` in `Model` menu
+    * To create a new model specify all model parameters and reaction equations
+      sequentially opening and filling tables in the `Model` menu (upper left
+      of the UI). Each table corresponds to a particular section of the
+      BioNetGen file of the model (with except for `Diffusions` and `Geometry`
+      section). All expressions should comply with BioNetGen language.
+#.  To add diffusion (needed if using STEPS) navigate to corresponding section
```

Comment:
I would exchange 'needed' by 'required' to be consistent with the next sentence

Reply:
Changed